src/configureStore.js

```diff
@@ -0,0 +1,5 @@
+import {applyMiddleware, createStore} from "redux";
+import {userReducer} from "./store/reducer";
+import thunkMiddleware from "redux-thunk"
+
+export const store = createStore(userReducer, applyMiddleware(thunkMiddleware));
```

Comment:
We can not to use redux yet :) You will be able to study this library later.
But in any case - it's your choise. If you are sure that you can do it, then continue